controllers/gatekeeper_controller.go

```diff
@@ -203,19 +220,24 @@ func (r *GatekeeperReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			"spec.image.image", gatekeeper.Spec.Image.Image)
 	}
 
+	if err := r.addConstraintController(ctx, gatekeeper); err != nil {
+		requeueTime = time.Second * 3
```

Comment:
This is an appropriate use of the controller-runtime exponential backoff
```suggestion
		return reconcile.Result{}, err
```